### DISCUSSION TITLE
scoring rules for these dietary ratings

### DISCUSSION THREAD

**Discussion by laohutie** [jamesjiadazhan/dietaryindex#213] [Q&A] [answered]
Hello teacher, may I ask where are the specific scoring rules for these dietary ratings? Because when I checked DASHI, I may not have been able to find a reasonable explanation for his scores due to my own reasons. I bothered the teacher. Thank you teacher.

**Comment by jamesjiadazhan** [comment]
Below is where you can find more about DASHI
https://github.com/jamesjiadazhan/dietaryindex/blob/main/Manual.md

<img width="726" alt="image" src="https://github.com/jamesjiadazhan/dietaryindex/assets/108076575/71fd4ccb-b1cc-4901-9f37-2d860e7e7266">


Let me know if you have other questions. If possible, please give our work a Star on GitHub and cite our work when you publish the work. It helps recognize and appreciate our intellectual efforts and contributions.

<img width="595" alt="image" src="https://github.com/jamesjiadazhan/dietaryindex/assets/108076575/ae60cae6-c09b-4a8d-93b3-e81ac776f9ba">

Jiada James Zhan, Rebecca A Hodge, Anne Dunlop, et al. Dietaryindex: A User-Friendly and Versatile R Package for Standardizing Dietary Pattern Analysis in Epidemiological and Clinical Studies. bioRxiv. Published online August 07, 2023:2023.08.07.548466. doi:10.1101/2023.08.07.548466

**Reply by laohutie** [reply]
Hello teacher, I have previously checked the relevant literature based on the reference. For DASHI, I learned that it is a dietary pattern suitable for NHANES scoring. However, for MEDI, I did not get a good explanation during the search process. What is MEDI? Is there a Mediterranean model rating specifically for consuming extra virgin olive oil? Thanks for the teacher’s answer!

**Reply by laohutie** [reply]
I have already given a star to the teacher’s Github project, and I’m going to trouble the teacher again.

**Reply by jamesjiadazhan** [reply]
MEDI is our customized MED score using absolute serving size from the PREDIMED study, not median-based. This helps to resolve the issue of incompatibility across different studies.

**Reply by laohutie** [reply]
Thank you again for your enthusiastic reply. I would also like to know if we have specific supplementary documents for everyone to understand the abbreviations of the calculated variables. In addition, I found during use: AHEI from 0-110, DASH from 8-40, DASHI From 0-9, HEI2016 from 0-100, MED-ALL from 0-9, MEDI-ALL from 0-9, DII-ALL from about -4.43-4.95, I don’t know if my judgment is correct, bother the teacher again, already Star the work and cite it in the future

**Reply by jamesjiadazhan** [reply, marked as answer]
Hi, please check the dietaryindex GitHub page to look for the Excel files that explain everything about all dietary index serving sizes and scores:

For a detailed explanation of these indexes, please check the attached Excel files:

[dietaryindex_SERVING_SIZE_DEFINITION.xlsx](https://github.com/jamesjiadazhan/dietaryindex/blob/main/dietaryindex_SERVING_SIZE_DEFINITION.xlsx)
[dietaryindex_SCORING_ALGORITHM.xlsx](https://github.com/jamesjiadazhan/dietaryindex/blob/main/dietaryindex_SCORING_ALGORITHM.xlsx)

**Comment by laohutie** [comment]
Sorry to bother the teacher. I wonder if the teacher has time to look at the student's problem. It has been bothering me for a long time and cannot be solved.

**Reply by jamesjiadazhan** [reply]
Please check the above reply. Sorry for the delay.

**Reply by laohutie** [reply]
thanks
